### PR TITLE
Optimization

### PR DESCRIPTION
**Pull request by mugishasam123**
- Add data in database which made it to slow down
- after we optimized it using indexing
- screenshots of actions are in readme file

# screenshots
- before

first

- after
first optimizer

- before
second

- after

second optimizer

- before
third

- after

third optimier

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,24 @@
 > This template should be used for database related projects at Microverse.
 > Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
 
+## Optimization
+After adding many millions of data in database it started to slow down and we decided to optimize it using indexing.
+
+# screenshots
+- before
+![screenshot](img/first.PNG)
+- after
```
Comment (SajjadAhmad14) [inline README.md:12]: The screenshot with the title `After` should have an execution time of less than 1000ms. Please add a screenshot that shows the performance increment after adding Index on foreign keys. You may remove indexes and add them once again to see if it helps to increase the query performance.

File: README.md [hunk 2]
```diff
@@ -3,6 +3,24 @@
...
+![screenshot](img/first%20optimizer.PNG)
+
+- before
+![screenshot](img/second.PNG)
+- after
```
Comment (SajjadAhmad14) [inline README.md:17]: The screenshot with the title `After` should have an execution time of less than 1000ms.

File: README.md [hunk 3]
```diff
@@ -3,6 +3,24 @@
 > This template should be used for database related projects at Microverse.
 > Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
 
+## Optimization
+After adding many millions of data in database it started to slow down and we decided to optimize it using indexing.
+
+# screenshots
```
Comment (SajjadAhmad14) [inline README.md:9]: Please add all these screenshots to the description of the Pull Request as it is requested in the project requirements.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi team, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
PR has a title and summary. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 3 above)
Comment (mugishasam123) [reply README.md:9]: its difficult for us to upload them in pr but hope reviewer can also see them in readme file

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi team, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**